app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md

```diff
@@ -22,8 +22,9 @@ keywords:
   - apply
   - how to apply
   - applying for teacher training
-promo_content:
-    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+cta_adviser:
+  adviser:
+    text: Maximise your chances of submitting a successful application with the support of a dedicated adviser with years of teaching experience. Chat by phone, text or email, as little or as often as you need.
```

Comment:
I think we could probably make this text a bit more relevant to the page here?

Reply:
```suggestion
    text: An adviser with years of teaching experience can help you understand why your application was unsuccessful and what you can do next. Chat by phone, text or email, as little or as often as you need.
```